modules/doc/content/getting_started/installation/offline_installation.md

```diff
@@ -1,52 +1,50 @@
+If you are using a machine (or HPC cluster) with no access to the internet, the following
+instructions will help you create an environment suitable for MOOSE-based development.
 
-Please be certain, that the machine in which you intend to perform the actual work (which this document will refer to as the 'target' machine), meets our minimum requirements:
+Please be certain, that the machine in which you intend to perform the actual work (which this
+document will refer to as the 'target' machine), meets the following requirements:
 
 !include sqa/minimum_requirements.md
 
-## Dependency List
+# Prerequisites
 
-Ultimately, we will need to build the following stack, to create an environment suitable for MOOSE-based development on the target machine:
+#### Air-Gapped HPC Clusters
 
-- GCC, Or LLVM +and+ GCC
-- MPI Wrapper
-- PETSc, libMesh, and MOOSE
+If the target machine is an HPC cluster, it is likely your cluster already has an MPI wrapper (and
+thus a GCC or LLVM compiler). Please contact your system administrator and ask them how to
+appropriately make use of your cluster's MPI wrapper and compiler. Please note, Intel compilers are
+not supported.
 
-## Prerequisites
+#### Personal Workstation
 
-Before you begin, know that you will first need to be operating on a machine with internet access.
+If the target machine is your personal workstation, that machine must sufficiently achieve the
+following:
 
-If you are on a Windows machine, please install [Ubuntu 20.04](https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71) from the Microsoft store, simply to have access to a Linux terminal. Once inside the terminal, install 'git' and 'curl', as that will be needed to clone the repositories involved in later steps (`sudo apt-get install git curl`).
+!include installation/manual_prereqs.md
 
-## GCC
+In addition, the target machine will need an MPI wrapper. We recommend one of the following:
 
-It is highly likely the target machine in which you will be operating on will already contain an acceptable version of GCC. Please contact your system administrator for instructions on how to use that systems compiler. If not, you will need to head on over to GNU's [GCC](https://gcc.gnu.org/) site, and follow their instructions on building a GCC compiler at or beyond the above listed minimum version.
+- [MPICH](https://www.mpich.org/)
 
-## LLVM (optional)
+- [OpenMPI](https://www.open-mpi.org/)
 
-While arguably a [better compiler](https://opensource.apple.com/source/clang/clang-23/clang/tools/clang/www/comparison.html) over traditional GCC, it is more difficult to build. Nevertheless, if you wish to make the attempt, head on over to [llvm.org](http://llvm.org/), to begin.
+!alert note title=Personal Air-Gapped Workstation
+Procuring the above on a personal air-gapped workstation will need to be your responsibility. We
+cannot instruct a means for bypassing such security.
```

Comment:
not sure what you mean here.
bypass the security of air-gapping? why are we "bypassing" ?

Reply:
I don't want to instruct people how to create an SSH reverse tunnel. Or how to use/bring in a thumb stick into an obviously required air-gapped secure network...

They will need to figure out how to install these prerequisites on their own. We can't help them `apt-get install`, or `xcode-select`, or use App Store, etc, behind a non-internet accessing computer. This statement basically says: Contact your help desk.

Reply:
I can reword it a bit? Maybe something less security-like?

```bash
!alert note title=Personal Air-Gapped Workstation
Procuring the above on a machine with no network access is beyond the scope of this document. Please work with your system administrator on satisfying the above prerequisites. 
```

Reply:
That works